oap-server/server-bootstrap/src/main/resources/application.yml

```diff
@@ -337,7 +337,9 @@ envoy-metric:
 prometheus-fetcher:
   selector: ${SW_PROMETHEUS_FETCHER:-}
   default:
+    active: ${SW_PROMETHEUS_FETCHER_ACTIVE:false}
     enabledRules: ${SW_PROMETHEUS_FETCHER_ENABLED_RULES:"self"}
+    # maxConvertWorker: ${SW_PROMETHEUS_FETCHER_NUM_CONVERT_WORKER:8}
```

Comment:
Commenting out this line is not a good idea, it makes the env var not work and the user have to edit this file. Please default this value to 0 and set it to cpuCore / 2 if it’s 0.

Reply:
0 or negative, should make the codes falling back to core / 2.